packages/cursorless-org-docs/src/docs/user/languages/components/ScopeSupportForLevel.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  camelCaseToAllDown,
+  capitalize,
+  groupBy,
+  type ScopeSupportFacet,
+  type ScopeSupportFacetInfo,
+  scopeSupportFacetInfos,
+  type ScopeType,
+  type SimpleScopeTypeType,
+} from "@cursorless/common";
+import React, { useState } from "react";
+
+interface Props {
+  facets: ScopeSupportFacet[];
+  title: string;
+  subtitle: string;
+  description?: React.ReactNode;
+  expanded?: boolean;
+}
+
+export function ScopeSupportForLevel({
+  facets,
+  title,
+  subtitle,
+  description,
+  expanded: expandedProp,
+}: Props): JSX.Element | null {
+  const [expanded, setExpanded] = useState(expandedProp ?? false);
+
+  if (facets.length === 0) {
+    return null;
+  }
+
+  const renderBody = () => {
+    if (!expanded) {
+      return null;
+    }
+
+    const facetInfos = facets.map(
+      (facet): AugmentedFacetInfo => ({
+        facet,
+        ...scopeSupportFacetInfos[facet],
+      }),
+    );
+    const scopeGroups: Map<string, AugmentedFacetInfo[]> = groupBy(
+      facetInfos,
+      (facetInfo) => serializeScopeType(facetInfo.scopeType),
+    );
+    const scopeTypes = Array.from(scopeGroups.keys()).sort();
+
+    return (
+      <div className="card-body">
+        {description && <p>{description}</p>}
+
+        {scopeTypes.map((scopeType) => {
+          const facetInfos = scopeGroups.get(scopeType) ?? [];
+          return (
+            <div key={scopeType}>
+              <h4>{prettifyScopeType(scopeType)}</h4>
+              <ul>
+                {facetInfos.map((facetInfo) => {
+                  return (
+                    <li key={facetInfo.facet}>
+                      <b title={facetInfo.facet}>
+                        {prettifyFacet(facetInfo.facet)}
+                      </b>
+                      : {facetInfo.description}
+                    </li>
+                  );
+                })}
+              </ul>
+            </div>
+          );
+        })}
+      </div>
+    );
+  };
+
+  return (
+    <div className="card">
+      <div className="card-header" onClick={() => setExpanded(!expanded)}>
+        <h3>{title}</h3>
+        {subtitle}
+      </div>
+
+      {renderBody()}
+    </div>
+  );
+}
+
+interface AugmentedFacetInfo extends ScopeSupportFacetInfo {
+  facet: ScopeSupportFacet;
+}
+
+function prettifyScopeType(scopeType: string): string {
+  return capitalize(camelCaseToAllDown(scopeType));
+}
+
+function prettifyFacet(facet: ScopeSupportFacet): string {
+  const parts = facet.split(".").map(camelCaseToAllDown);
+  if (parts.length === 1) {
+    return capitalize(parts[0]);
```

Comment:
Some facets just have the same name as the scope. In that case I just keep it.

![image](https://github.com/user-attachments/assets/b958c4b4-1a03-4de9-97ad-d86268eb0d07)
![image](https://github.com/user-attachments/assets/8f988795-b48e-4ff9-9814-1c76f588938e)